src/helper.js

```diff
@@ -20,6 +25,34 @@ Helper.config = require(path.resolve(path.join(
 	"config.js"
 )));
 
+function getVersion() {
+	var gitRef = getGitRef();
+	if (gitRef) {
+		return gitRef.replace(/^v/, "");
+	}
+	return pkg.version;
+}
+
+let _gitRef;
+function getGitRef() {
+	if (_gitRef) {
+		return _gitRef;
+	}
+	const commands = ["git describe HEAD", "git rev-parse --short HEAD"];
```

Comment:
So, if the first command succeeds, the `for` loop assigns stuff then continues to the next one, but if it fails, it... continues to the next one? 😅
Not sure that was the intent here.

Indeed, you can go with just one command, no need to over-complexify the code for this. Chances that the command is not available are way lower than not having `git` at all, and it's fine to default to npm version otherwise.